tests/test_gin.py

```diff
@@ -90,6 +98,17 @@ class TestNwbConversions(unittest.TestCase):
                 )
             )
 
+        if platform != "darwin" or version.parse(python_version()) >= version.parse("3.8"):
+            ced_file_path = str(DATA_PATH / "spike2" / "m365_1sec.smrx")
+            channel_info = CEDRecordingInterface.get_all_channels_info(file_path=ced_file_path)
+            rhd_channels = [ch for ch, info in channel_info.items() if "Rhd" in info["title"]]
+            parameterized_recording_list.append(
+                param(
+                    recording_interface=CEDRecordingInterface,
+                    interface_kwargs=dict(file_path=ced_file_path, smrx_channel_ids=rhd_channels),
+                ),
+            )
+
```

Comment:
ok, so from here I understand we're not testing with MacOS, right? The tests Python v is 3.7
is this intended?

we can probably get rid of the `class test(TestCase)` in `tests/test_interfaces.py` then?

Reply:
MacOS does run full tests, yes (when GIN data available). See https://github.com/catalystneuro/nwb-conversion-tools/pull/307 for v3.8 and 3.9